python/ml4ir/base/config/keys.py

```diff
@@ -31,6 +31,19 @@ class OptimizerKey(Key):
     SGD = "sgd"
     RMS_PROP = "rms_prop"
 
+class LearningRateScheduleKey(Key):
+    """Learning rate schedule keys"""
+
+    EXPONENTIAL = "exponential"
```

Comment:
@mohazahran Add a default/uniform learning rate schedule as well.

Reply:
I'm using a 'constant' schedule if none was provided by setting the decay of the exponential schedule to 1.0
Should I add a 'constant' schedule as an option?

Reply:
I think so @mohazahran . You are right, you are doing it. But it is hack-y. You can rightfully claim it was done before! But before we tried to control it with argparser args, so the idea was to minimize as much as possible their number. now that you are decoupling the design (push it to configs) and the implementation (optimizers + schedulers get their respective functions) I think it makes sense to have a 3rd one. For the sake of argument you could also argue that a Cyclic LR with the same min and max lr is a constant!